view-creator-framework/src/main/java/org/hypertrace/core/viewcreator/pinot/PinotViewCreatorConfig.java

```diff
@@ -57,4 +57,11 @@ public class PinotViewCreatorConfig {
   // REST API
   /////////////
   public static final String PINOT_REST_URI_TABLES = "tables";
+
+  // text index config keys
+  public static final String TEXT_INDEX_CONFIG_COLUMN = "column";
```

Comment:
should we move them to generic field configs? 
just rethinking as it will be a one to one mapping between traceable spec and pinot table config sepc.

Reply:
Done.
Align with :  https://docs.pinot.apache.org/configuration-reference/table#field-config-list